src/clj/rems/api.clj

```diff
@@ -101,7 +123,10 @@
   request)
 
 (def api-routes
-  (api
+  ;; we wrap audit-log-middleware outside compojure-api since we want
+  ;; to see the status codes produced by our exception handlers
+  (audit-log-middleware
```

Comment:
I wonder if it should be a config option to audit log or not?

Reply:
Possibly need to implement some configurable deleting / vacuuming of old data.

Reply:
Cleaning data is luckily easy to do on the DB level. Let's wait for what the users need. I'm a bit hesitant to add auto-cleaning to REMS, as e.g. a system clock glitch might wipe the whole audit log.

Reply:
It could be provided as an ops script sometime later.